elk/training/ccs.py

```diff
@@ -141,31 +118,29 @@ def fit(
         if verbose:
```

Comment:
Should we maybe remove verbose also from the css class, and just always print the things under the ifs ?

Reply:
Hmm I think we should either keep `verbose` or _never_ print, because I want to support programmatic uses where you may not want to print every time you call `fit`. Like imagine if PyTorch optimizers always printed when you initialized them or called `step` or something.